doc/source/user_guide/groupby.rst

```diff
@@ -537,41 +574,84 @@ Another aggregation example is to compute the number of unique values of each gr
-Some common aggregating functions are tabulated below:
 
-.. csv-table::
-    :header: "Function", "Description"
-    :widths: 20, 80
-    :delim: ;
+.. _groupby.aggregate.agg:
+
+The :meth:`~.DataFrameGroupBy.aggregate` method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    The :meth:`~.DataFrameGroupBy.aggregate` method can accept many different types of
+    inputs. This section details using string aliases for various GroupBy methods; other
+    inputs are detailed in the sections below.
+
+Any reduction method that pandas implements can be passed as a string to
+:meth:`~.DataFrameGroupBy.aggregate`. Users are encouraged to use the shorthand,
+``agg``. It will operate as if the corresponding method was called.
 
-        :meth:`~pd.core.groupby.DataFrameGroupBy.mean`;Compute mean of groups
-        :meth:`~pd.core.groupby.DataFrameGroupBy.sum`;Compute sum of group values
-        :meth:`~pd.core.groupby.DataFrameGroupBy.size`;Compute group sizes
-        :meth:`~pd.core.groupby.DataFrameGroupBy.count`;Compute count of group
-        :meth:`~pd.core.groupby.DataFrameGroupBy.std`;Standard deviation of groups
-        :meth:`~pd.core.groupby.DataFrameGroupBy.var`;Compute variance of groups
-        :meth:`~pd.core.groupby.DataFrameGroupBy.sem`;Standard error of the mean of groups
-        :meth:`~pd.core.groupby.DataFrameGroupBy.describe`;Generates descriptive statistics
-        :meth:`~pd.core.groupby.DataFrameGroupBy.first`;Compute first of group values
-        :meth:`~pd.core.groupby.DataFrameGroupBy.last`;Compute last of group values
-        :meth:`~pd.core.groupby.DataFrameGroupBy.nth`;Take nth value, or a subset if n is a list
-        :meth:`~pd.core.groupby.DataFrameGroupBy.min`;Compute min of group values
-        :meth:`~pd.core.groupby.DataFrameGroupBy.max`;Compute max of group values
-
-
-The aggregating functions above will exclude NA values. Any function which
-reduces a :class:`Series` to a scalar value is an aggregation function and will work,
-a trivial example is ``df.groupby('A').agg(lambda ser: 1)``. Note that
-:meth:`~pd.core.groupby.DataFrameGroupBy.nth` can act as a reducer *or* a
-filter, see :ref:`here <groupby.nth>`.
+.. ipython:: python
+
+   grouped = df.groupby("A")
+   grouped[["C", "D"]].aggregate("sum")
+
+   grouped = df.groupby(["A", "B"])
+   grouped.agg("sum")
+
+The result of the aggregation will have the group names as the
+new index along the grouped axis. In the case of multiple keys, the result is a
+:ref:`MultiIndex <advanced.hierarchical>` by default. As mentioned above, this can be
+changed by using the ``as_index`` option:
+
+.. ipython:: python
+
+   grouped = df.groupby(["A", "B"], as_index=False)
+   grouped.agg("sum")
+
+   df.groupby("A", as_index=False)[["C", "D"]].agg("sum")
+
+Note that you could use the ``reset_index`` DataFrame function to achieve the
+same result as the column names are stored in the resulting ``MultiIndex``:
```

Comment:
Notice instead of Note; since instead of as.

Reply:
I think both of these are merely a matter of personal taste / style and don't think they are any better or worse than what is there.

Reply:
I read that "Note" suggests the person will actually write something about it. But it's clear what you mean. Plus I can't find were I read it!... So I'm good.